comps/third_parties/nebula/deployment/kubernetes/community_edition.yaml

```diff
@@ -0,0 +1,75 @@
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+apiVersion: apps.nebula-graph.io/v1alpha1
+kind: NebulaCluster
+metadata:
+  name: nebula
+  #namespace: nebula-operator-system
+spec:
+  graphd:
+    resources:
+      requests:
+        cpu: "500m"
+        memory: "500Mi"
+      limits:
+        cpu: "1"
+        memory: "1Gi"
+    replicas: 1
+    image: vesoft/nebula-graphd
```

Comment:
should we pin the image tag instead of using latest image? Just in case future versions of nebula might not be compatible with this resource operator definition.